### DISCUSSION TITLE
How to import redis rdb file to kvrocks?

### DISCUSSION THREAD

**Discussion by seaguest** [apache/kvrocks#591] [General]
Hello,

I want to switch to kvrocks from redis, I tried to find a way to import data from rdb but I can't find any documentation on that.

How can I do that?

**Comment by git-hulk** [comment]
@seaguest There're two ways to achieve this:

1. Use the RDB tool to translate into AOF, then imports the data with `redis-cli`
2. Sutup the Redis instance with the RDB file and use the migration tool like `https://github.com/vipshop/redis-migrate-tool`

**Comment by cloudufull** [comment]
https://github.com/vipshop/redis-migrate-tool 支持的redis版本太低了，，
